### PR TITLE
Adding contributing.md (#7)

### PR DESCRIPTION
**Pull request by Anoninnyc**
Bare bones contributing.md addressing #7 , specifically GIT workflow.
Pls comment

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# GIT workflow
+
+One-click solution for personalized results.
+
+## Steps
+* Each dev forks & clones Master repo.
+* Any new features should be on it's own branch w/ the following naming convention:
```
Comment (mLuby) [inline CONTRIBUTING.md:5]: on its own

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+ ####lastName/feature/issue-tracker-number
+ (*Ex. Arya/userAuth/#54*)
+
+* After completing a feature, commit and push to corresponding branch on your forked repo, ensuring it passes a Travis CI build in the process.
+* Create PR.
+* Have at least 2 peers approve PR before merge can be made [by 2nd reviewer].
```
Comment (mLuby) [inline CONTRIBUTING.md:12]: I'm not a fan of this step for 2 reasons. 1: github can only enforce one review. 2: merge by reviewer has in my experience been much slower than merge by submitter.

**Review by mLuby — APPROVED** [review]
Thanks for adding this. My comments are non-blocking so feel free to merge.

File: CONTRIBUTING.md (hunk 1 above)
Comment (aeb0) [reply CONTRIBUTING.md:5]: any reason for having devs fork the repo?  not necessarily against it but also don't see the benefit

Comment (Anoninnyc) [reply CONTRIBUTING.md:5]: @aebrow4
That proposed workflow feature is a trait of https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow

I'm open to any pattern, though I've found that having everybody push directly to master/source of truth could get messy and/or result in accidental writes to master branch...

**Pull request merged**